packages/machine-emulator-tools/Dockerfile

```diff
@@ -6,6 +6,5 @@ ADD https://github.com/cartesi/machine-emulator-tools/releases/download/v${VERSI
 RUN mkdir -p /tmp/machine-emulator-tools && tar -C /tmp/machine-emulator-tools -xzf /tmp/machine-emulator-tools.tar.gz
 
 FROM scratch as tools
-COPY --from=download /tmp/machine-emulator-tools/sbin/init /usr/sbin/init
 COPY --from=download /tmp/machine-emulator-tools/etc/hostname /etc/hostname
 COPY --from=download /tmp/machine-emulator-tools/opt/cartesi /opt/cartesi/
```

Comment:
looks like `/opt/cartesi` is being deprecated in favor os system's standard paths

should we change this to keep it consistent?

Reply:
This is still like it in the tar.gz.
Actually we need to change this to just copy everything to `/`